src/devices/Pcd8544/Pcd8544.csproj

```diff
@@ -7,6 +7,8 @@
 
   <ItemGroup>
     <Compile Include="*.cs" />
+    <Compile Include="..\Common\Iot\Device\Graphics\BdfFont.cs" />
```

Comment:
Same here. Don't include files from other tree locations directly (might cause problems if another project needs to import this one)